src/Xrefcheck/Verify.hs

```diff
@@ -682,8 +689,11 @@ parseUri link = do
 checkExternalResource :: [Text] -> Config -> Text -> IO (VerifyResult VerifyError)
 checkExternalResource followed config@Config{..} link
   | isIgnored = return mempty
+  | link `elem` followed = pure $
+      VerifyResult [RedirectChainCycle (link :| followed)]
+  | ncMaxRedirectFollows >= 0 && length followed > ncMaxRedirectFollows = pure $
```

Comment:
Which purpose this `ncMaxRedirectFollows >= 0`  serves?

Reply:
It is because I allowed to disable the limit when given a negative value. Do you see it useful? For example:

- Limit set to 2: follow 2 redirects at most.
- Limit set to 0: no redirect following allowed.
- Limit set to -1: no limit in the redirect following.